TRAVIS

```diff
@@ -1,6 +1,13 @@
 Implement continuous integration
 
-Implement a track test suite that can run both locally and on Travis CI. The track test suite should verify that each exercise makes sense, by running the exercise tests against the example solution.
+Implement the "track test suite".  This suite should:
+
+- for each exercise in the track,
+  - run that exercise's test suite (aka the "exercise test suite");
+  - use the exercise's solution as the code under test.
+- ensure all test cases are enabled;
```

Comment:
Noice.  In fact, I think it would best serve as a sub-bullet of "run that exercise's test suite".